benchmark/camelyon/workflows.py

```diff
@@ -120,14 +127,15 @@ def accuracy(data_from_opener, predictions):
 
     # Launch experiment
     compute_plan = execute_experiment(
-        client=clients[1],
+        client=clients[0],
```

Comment:
Maybe add a comment why we are choosing client 0 ? (I assume it is due to the builder ?)

Reply:
It is indeed!